Modules/Karma.cs

```diff
@@ -18,7 +18,7 @@ public class Karma: IModule
 		private const string KarmaDisabledString = "Karma is disabled on this server.";
 
 		private ValkyrjaClient Client;
-		private readonly Regex RegexKarma = new Regex(".*(?<!\\\\)(thank(?!sgiving)|thx|ʞuɐɥʇ|danke|vielen dank|gracias|merci(?!al)|grazie|arigato|dziękuję|dziekuje|obrigad).*", RegexOptions.Compiled | RegexOptions.IgnoreCase);
+		private readonly Regex RegexKarma = new Regex(".*(?<!\\\\)(thank(?!sgiving)|thx|ʞuɐɥʇ|danke|vielen dank|gracias|merci(?!al)|grazie|arigato|dziękuję|dziekuje|obrigad|(?:^|\W)ty(sm)?(?:$|\W)).*", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
the non-sub string matching here could be extended to all other words in this regex.

Reply:
Honestly we would need to think about grammar for all the languages - nonenglish languages modify the word with grammar, often appending or prepending things. And that's a lot of effort.

Reply:
I think the easier solution might be for you to use some translation library/api to convert everything to English or any other simpler language containing only one way to say thanks. then check the word after translation